glib/src/types.rs

```diff
@@ -246,6 +246,18 @@ impl StaticType for Type {
     }
 }
 
+pub trait StaticTypeExt {
+    /// Ensures that the type has been registered with the type system.
```

Comment:
You need the rustdoc-stripper-ignore-next comment here

Reply:
Done